modules/catalog-compaction/src/main/java/org/apache/ignite/internal/catalog/compaction/CatalogCompactionRunner.java

```diff
@@ -211,12 +203,24 @@ public void updateCoordinator(ClusterNode newCoordinator) {
         triggerCompaction(lowWatermark);
     }
 
-    /** Returns local view of the node on who is currently compaction coordinator. For test purposes only.*/
+    /** Returns local view of the node on who is currently compaction coordinator. For test purposes only. */
     @TestOnly
     public @Nullable String coordinator() {
         return compactionCoordinatorNodeName;
     }
 
+    /** Returns {@link MinimumRequiredTimeCollectorService}. For test purposes only. */
+    @TestOnly
+    public MinimumRequiredTimeCollectorService localMinTimeCollectorService() {
+        return localMinTimeCollectorService;
+    }
```

Comment:
so, you've introduced TestOnly method to expose object that was created outside and passed in constructor of CompactionRunner just one line before in the test... please remove this method

Reply:
Fixed.